emva1288/process/plotting.py

```diff
@@ -194,6 +194,93 @@ def plot(self, test):
         self.set_legend(ax)
 
 
+class PlotUyDarkCurrent(Emva1288Plot):
+    name = 'Dark Current From Mean'
+    xlabel = 'exposure time [ns]'
+    ylabel = r'$\mu_{y.dark}$ [DN]'
+
+    def plot(self, test):
+        ax = self.ax
+
+        if len(test.darkcurrent['texp']) > 2:
+            data = test.darkcurrent
+        else:
+            data = test.temporal
+
+        if test.dark_current_fit_mean() is not np.nan:
+            X = data['texp']
+
+            ax.plot(X,
+                    data['u_ydark'],
+                    marker='o',
+                    markersize=5,
```

Comment:
This marker style makes it a little difficult to track deviation from fit, I think sticking to default plotting params for the data is fine
```suggestion
```
![image](https://github.com/EMVA1288/emva1288/assets/47544214/0a1fda52-d1b2-4a48-b803-435efdbd340f)